src/com/javabasic/lesson08/homework/Task01.java

```diff
@@ -0,0 +1,106 @@
+package com.javabasic.lesson08.homework;
+
+public class Task01 {
+
+    public static final int minAgePlayers = 18;
+    public static final int maxAgePlayers = 45;
+    public static final int numberOfPlayers = 11;
+
+    public static void main(String[] args) {
+
+        Integer[][] teams = new Integer[][]{
+                new Integer[numberOfPlayers],
```

Comment:
you can use primitive type `int` here, use it always, as it more lightweight, switch to Integer wrapper only if you have reason for that